ARAnalytics.m

```diff
@@ -620,3 +632,5 @@ void ARAnalyticsEvent (NSString *event, NSDictionary *properties) {
 const NSString *ARLibratoEmail = @"ARLibratoEmail";
 const NSString *ARLibratoToken = @"ARLibratoToken";
 const NSString *ARLibratoPrefix = @"ARLibratoPrefix";
+const NSString *ARIntercomApiKey = @"ARInteSetuprcomApiKey";
```

Comment:
this should be `NSString *const` = it is a pointer to a constant
This is the way pointer constants are defined in objc (e.g. check out `UIApplicationDidBecomeActiveNotification`)